pyproject.toml

```diff
@@ -2,7 +2,7 @@
 name = "nautobot"
 # Primary package version gets set here. This is used for publishing, and once
 # installed, `nautobot.__version__` will have this version number.
-version = "1.2.7"
+version = "1.2.8-beta.1"
```

Comment:
Reminder to update to "1.2.8" before merge.

Reply:
💡